src/test/resources/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImplFulfilmentTest.CaseUpdate.json

```diff
@@ -0,0 +1,56 @@
+[
+	{
+		"caseId": "dc4477d1-dd3f-4c69-b181-7ff725dc9fa4",
+		"surveyId": "a66de4dc-3c3b-11e9-b210-d663bd873d93",
+		"collectionExerciseId": "a66de4dc-3c3b-11e9-b210-d663bd873d93",
+		"sample": {
+			"addressLine1": "1 main street",
+			"addressLine2": "upper upperingham",
+			"addressLine3": "",
+			"townName": "upton",
+			"postcode": "UP103UP",
+			"region": "E",
+			"uprn": "123456"
+		},
+		"sampleSensitive": {
+			"phoneNumber": "REDACTED"
+		},
+		"invalid": false
+	},
+	{
```

Comment:
not a lot of point in having more than one object since the others are not used in the test.